.eslintrc.json

```diff
@@ -2,7 +2,8 @@
     "extends": "eslint:recommended",
     "root": true,
     "env": {
-        "node": true
+        "node": true,
+        "es6": true
```

Comment:
Added this line for Set support, as per eslint [docs](https://eslint.org/docs/latest/use/configure/language-options#specifying-parser-options):

> In summary, to support only ES6 syntax, use { "parserOptions": { "ecmaVersion": 6 } }, and to support both ES6 syntax and new ES6 global variables, such as Set and others, use { "env": { "es6": true } }.

Reply:
This above doc seems slightly wrong, as after adding this line and removing the `ecmaVersion` under `parserOptions`, we started seeing some strange syntax errors. Fixed by putting the `ecmaVersion` option back: https://stackoverflow.com/questions/50285821/eslint-parsing-error-unexpected-token-function-with-async